src/Commands/BuildCommand.php

```diff
@@ -161,6 +161,10 @@ private function compile(string $name): BuildCommand
         $this->task('   2. <fg=yellow>Compile</> into a single file');
 
         $this->output->newLine();
+        
+        if (! File::exists($this->app->basePath($this->getBinary()) . '.phar')) {
+            throw new \RuntimeException('Failed to compile the application.');
+        }
 
         File::move($this->app->basePath($this->getBinary()).'.phar', $this->app->buildsPath($name));
```

Comment:
Can we inline the result of the path to reduce duplication of the method calls? 🙂
```suggestion
        $pharPath = $this->app->basePath($this->getBinary()) . '.phar';
        
        if (! File::exists($pharPath)) {
            throw new RuntimeException('Failed to compile the application.');
        }

        File::move($pharPath, $this->app->buildsPath($name));
```

Can we also import `RuntimeException` at the top of the file? 🙂